tests/ports/psoc6/hw_ext/adc.py

```diff
@@ -16,21 +16,25 @@
 # Allocate pin based on board
 board = os.uname().machine
 if "CY8CPROTO-062-4343W" in board:
+    gnd_tests_skip = False
     adc_pin_gnd = "P10_4"
     adc_pin_mid = "P10_3"
     adc_mid_chan = 3
     adc_pin_max = "P10_2"
     adc_wrong_pin_name = "P13_7"
 elif "CY8CPROTO-063-BLE" in board:
+    gnd_tests_skip = False
     adc_pin_gnd = "P10_2"
     adc_pin_mid = "P10_3"
     adc_mid_chan = 3
     adc_pin_max = "P10_4"
     adc_wrong_pin_name = "P13_7"
 elif "CY8CKIT-062S2-AI" in board:
-    # P10_x only available in expansion header J17 and only 2 pins
-    print("SKIP")
-    raise SystemExit
+    gnd_tests_skip = True
```

Comment:
Gnd test does not work is it?

Reply:
Yes, it does. The problem is that we only have 2 ADC capable pins accessible in the pin header of the AI Kit. 
As the .exp is printing gnd, vdd/2 and vdd, this board either dummy prints a missing ADC signal or... we consider another approach for this test.

And alternative I came up with is to print results only in case of failure. Then the .exp file is silent, but still the failures would be reported and we don´t miss the debugging capabilities. Therefore the previous boards, which have 3 pins available can tests 3 cases, and the AI board 2 adc inputs only.